cards/src/main/resources/cards/custom/group0/other_death_knight/koft/spell_heartstop_aura.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Heartstop Aura",
+  "baseManaCost": 3,
+  "type": "SPELL",
+  "heroClass": "SPIRIT",
+  "rarity": "EPIC",
+  "description": "Your minions can't be destroyed until your next turn.",
```

Comment:
It's hard to say what the intention of this card is. Can't be destroyed is really a fishy mechanic. It probably needs to go to staging until we get an alternative implementation.

I'd rather the card say, simply, "Your minions are immune until your next turn," and make it cost a lot more. Or "Your minions' health can't be reduced below 1 until your next turn." Or "Until the start of your next turn, your minions will be resurrected." All of these are a lot more clear.

I'm just not sure what it means for the minion's health.